cmd/root.go

```diff
@@ -119,10 +120,19 @@ func rootCmd(o *options.Options) error {
 		return fmt.Errorf("GetEnabled: %w", err)
 	}
 
-	if o.Format == options.FormatDefault {
-		for checkName := range enabledChecks {
-			fmt.Fprintf(os.Stderr, "Starting [%s]\n", checkName)
+	// Define output to console or file
+	var output io.Writer = os.Stdout
+	if o.Output != "" {
+		outputF, err := os.Create(o.Output)
+		if err != nil {
+			return fmt.Errorf("unable to create output file: %w", err)
 		}
+		defer outputF.Close()
+		output = outputF
```

Comment:
do we need an intermediate var?

Reply:
No, we don't, I can create a function to wrap this.

Reply:
oh i just meant not using `:=`, not necessarily a helper but that works too
```go
	// Define output to console or file
	var output io.Writer = os.Stdout
	if opts.Output != "" {
		output, err = os.Create(opts.Output)
		if err != nil {
			return fmt.Errorf("unable to create output file: %w", err)
		}
		defer output.Close()
	}
```

Reply:
ah okay! got it. i will pay attention for that next time.